src/parser/hunter/beastmastery/modules/core/SpellUsable.tsx

```diff
@@ -6,12 +6,19 @@ class SpellUsable extends CoreSpellUsable {
     ...CoreSpellUsable.dependencies,
   };
 
-  lastPotentialTriggerForBarbedShotReset = null;
-  on_byPlayer_cast(event) {
+  lastPotentialTriggerForBarbedShotReset: {
+    timestamp?: any;
+    ability?: {
+      guid: any;
+    };
+  } | null | undefined;
+  on_byPlayer_cast(event: { ability: { guid: any; }; } | null) {
```

Comment:
```suggestion
  on_byPlayer_cast(event: CastEvent) {
```

This should never be able to be null?

Reply:
![image](https://user-images.githubusercontent.com/29204244/73134376-1c928f00-4036-11ea-9e09-1658465ed199.png)

Reply:
Fixes when changed to CastEvent though 😄